### PR TITLE
Add wording definitions

### PR DESCRIPTION
**Pull request by curquiza**

### CODE REVIEW

**Review by eskombro — COMMENTED** [review]
🌮

File: README.md [hunk 1]
```diff
@@ -23,6 +23,16 @@
   <a href="https://github.com/meilisearch/MeiliSearch/discussions" alt="Discussions"><img src="https://img.shields.io/badge/github-discussions-red" /></a>
 </p>
 
+The Integration Team owns all the repositories and tools that help the users integrate MeiliSearch in their project.
+
+Here are the different groups of integrations:
+- the **SDKs**: the *devkits* that help the devs to use MeiliSearch with their favorite language.
+- the **Front-End Integrations**: to integrate a smooth search experience in the front-end of the projects.
+- the **Web Framework Integrations**: to connect the web frameworks with MeiliSearch.
```
Comment (bidoubiwa) [inline README.md:31]: What does web framework refers to?

Comment (curquiza) [reply README.md:31]: Symphony, rails, laravel, django...
Frameworks that provide a complete web environment for their application (back and front)

Comment (bidoubiwa) [reply README.md:31]: Why not just frameworks? As it is not mandatory that these will be used in a web environment (local application or intranets).

Comment (bidoubiwa) [reply README.md:31]: Okey! Indeed. [Web framework](https://en.wikipedia.org/wiki/Web_framework) is more precise and also more accurate than just `framework`.

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥

**Review by eskombro — APPROVED** [review]
Thanks :)

**Pull request merged**